lib/logger/test/logger_test.exs

```diff
@@ -89,14 +89,35 @@ defmodule LoggerTest do
   test "level/0" do
     assert Logger.level() == :debug
 
+    Logger.configure(level: :all)
+    assert Logger.level() == :debug
+
     Logger.configure(level: :info)
     assert Logger.level() == :info
 
+    Logger.configure(level: :notice)
+    assert Logger.level() == :info
+
     Logger.configure(level: :warn)
     assert Logger.level() == :warn
 
+    Logger.configure(level: :warning)
+    assert Logger.level() == :warn
+
     Logger.configure(level: :error)
     assert Logger.level() == :error
+
+    Logger.configure(level: :critical)
+    assert Logger.level() == :error
+
+    Logger.configure(level: :alert)
+    assert Logger.level() == :error
+
+    Logger.configure(level: :emergency)
+    assert Logger.level() == :error
+
+    Logger.configure(level: :none)
+    assert Logger.level() == :error
```

Comment:
This behaviour can be a little bit confusing, but I do not know how to handle it as otherwise it would be a breaking change.

Reply:
I think we should include `:all` and `:none` as official levels and make `Logger.level` simply return the Erlang level directly.

Reply:
What about `:warn`? It would change behaviour (however I do not know if there is anything relying on specific value being returned from `Logger.level()`).

Reply:
I think it is fine to change `:warn` because if anyone was matching on Logger.level and expecting specific results, they need to handle new outputs now anyway.